elpis/endpoints/model.py

```diff
@@ -141,6 +150,77 @@ def results():
     return jsonify({"status": 200, "data": data})
 
 
+@bp.route("/download", methods=["GET", "POST"])
+def download():
+    """Downloads the model files to the frontend"""
+    model: HFTModel = app.config["CURRENT_MODEL"]
+    if model is None:
+        logger.error("No current model exists")
+        return jsonify(MISSING_MODEL_RESPONSE)
+
+    zipped_model_path = Path("/tmp", "model.zip")
+    # zipped_model_path.mkdir(parents=True, exist_ok=True)
```

Comment:
I think we either delete the comment or mkdir with `zipped_model_path.parent` instead of the path itself. I think I remember that uncommenting this line as is would have resulted in an error where model.zip is now a directory.

Reply:
@aviraljain99 looks like this hasn't been resolved, can you check please?

Reply:
I think I am going to go with deleting the comment!